src/components/FiltersDrawer.tsx

```diff
@@ -93,6 +84,7 @@ export const FiltersDrawer = ({ open, onClose }: FiltersDrawerProps) => {
             value={filters?.level}
             options={transformOptions(levelMap)}
             onSelect={(level) => setFilters((prev) => ({ ...prev, level }))}
+            clearValue={() => setFilters((prev) => ({ ...prev, level: undefined }))}
```

Comment:
This is used everywhere, it could be a function like
```
const clearKey = (key: keyof StateType) => {
    setFilters((prev) => ({ ...prev, [key]: undefined }))
  }
```